sdk/containerregistry/container-registry/swagger/containerregistry.json

```diff
@@ -1599,9 +1533,7 @@
           "os": "windows",
           "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
           "configMediaType": "application/vnd.docker.container.image.v1+json",
-          "tags": [
-            "4.7.2-20180905-nanoserver-1803"
-          ],
+          "tags": ["4.7.2-20180905-nanoserver-1803"],
           "changeableAttributes": {
             "deleteEnabled": true,
             "writeEnabled": true,
```

Comment:
Would it be possible to make `ManifestAttributesBase.mediaType` and `ManifestAttributesBase.configMediaType` internal for preview 1?  They are part of the push/pull story which we're planning to work on in Preview 2.

Reply:
I marked them with `"x-accessibility": "internal",`